torch/_inductor/codegen/cuda/cuda_cpp_scheduling.py

```diff
@@ -0,0 +1,216 @@
+import logging
+from typing import cast, List
+
+from ... import config, ir
+from ...codecache import code_hash, get_path
+from ...ir import ComputedBuffer, CUDATemplateBuffer, Pointwise
+from ...scheduler import (
+    BaseSchedulerNode,
+    BaseScheduling,
+    FusedSchedulerNode,
+    Scheduler,
+    SchedulerNode,
+)
+from ...utils import get_fused_kernel_name, get_kernel_metadata, sympy_product
+from ...virtualized import V
+from ..common import IndentedBuffer
+
+from .cutlass_epilogue_gen import CUTLASSEVTOpNotImplementedError
+
+log = logging.getLogger(__name__)
+_cuda_epilogue_fusion_counter: int = (
+    0  # used by unit tests to verify number of fused epilogues
+)
```

Comment:
Rather than adding a new var, we could reuse the existing counters structure similar to:
https://github.com/pytorch/pytorch/blob/7cbf9869d56f78be5b5ac291b94a01ade34cfead/torch/_inductor/fx_passes/fuse_attention.py#L29

Reply:
done